### DISCUSSION TITLE
can't upload a .m4a file in post method

### DISCUSSION THREAD

**Discussion by AjayLingam7136** [encode/django-rest-framework#9153] [General]
FileNotFoundError at /api-v1/videoai/create
[Errno 2] No such file or directory: '/tmp/tmpqqzz9qg2.upload.m4a'

but works for other formats, why?

**Comment by sarathak** [comment]
@AjayNiven please provide more details

**Reply by AjayLingam7136** [reply]
### Api endpoint code
```
class CreateObject(APIView):
    
    def post(self, request):
        serializer = NewObjectSerializer(data=request.data)
        try:
            if serializer.is_valid(raise_exception=True):
                data = serializer.validated_data
                audio_path = request.FILES.get('audio', False)
            loved_one = Sample.objects.create(
                    user = request.user,
                    title=data['title'],
                    audio = audio_path,
                    picture = data['picture'],
                    text = data['text'],
                    source = source
                )
```
 upload file - sample.m4a (type - audio/mp4)

While posting the file .m4a file format alone getting these error but working for all other formats of audio.

FileNotFoundError at /api-v1/videoai/create
[Errno 2] No such file or directory: '/tmp/tmpqqzz9qg2.upload.m4a'
